samples/brokers/zeroconf/.gitignore

```diff
@@ -0,0 +1 @@
+/target
```

Comment:
is this needed?

Reply:
It is not.

Fixed: https://github.com/deislabs/akri/pull/163/commits/91def2a5f886cd36ca208c63d9baa77df725e008